filament/test/filament_test.cpp

```diff
@@ -66,7 +66,7 @@ static bool vec3eq(float3 a, float3 b) {
 }
 
 TEST(FilamentTest, AabbMath) {
-    constexpr Aabb aabb = {{4, 5, 6}, {12, 14, 11}};
+    Aabb aabb = {{4, 5, 6}, {12, 14, 11}};
```

Comment:
try `static constexpr`, if doesn't work use `static const`. Tried this with gcc 12.1 and it's working.